packages/xchain-thorchain/__tests__/client.test.ts

```diff
@@ -161,6 +161,10 @@ describe('Client Test', () => {
         node: 'new mainnet client',
         rpc: 'new mainnet client',
       },
+      stagenet: {
```

Comment:
Please add a test for `validateAddress` using `stagenet` ([line 152](https://github.com/xchainjs/xchainjs-lib/pull/456/files#diff-e33969a1dd498f30c36ef473117bb07d3244b1783aac289ab366ec292e4255bcL152))

Reply:
Test added. Resolved.